src/components/icon-button/icon-button.spec.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import 'jest-styled-components';
```

Comment:
This can be removed as it's imported globally

Reply:
resolved